templates/register.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Регистрация</title>
+</head>
+    <body>
+    <h1>Регистрация</h1>
```

Comment:
Тогда тут нужно добавить
```suggestion
    <h1>Регистрация</h1>
    {% with messages = get_flashed_messages() %}
    {% if messages %}
            <ul>
                {% for message in messages %}
                    <li>{{ message }}</li>
                {% endfor %}
            </ul>
        {% endif %}
    {% endwith %}
```